### PR TITLE
Update Live demo for classes, style and props

### PR DESCRIPTION
**Pull request by phoskee**
Hi guys!!!
For the time being, it allows some items to be selected and played with.
I used @ui.refreshable, which though creates a simple 'div' container. 
If this is the right method, how do I access that created 'div' to set a 'w-full'?

[Check this out for mor details!](https://discord.com/channels/1089836369431498784/1097852086164668558/1099448570635096197)

### CODE REVIEW

File: nicegui/elements/input.py [hunk 1]
```diff
@@ -51,6 +53,20 @@ def toggle_type(_):
 
         self.validation = validation
 
+        if autocomplete is not None:
+            def AutoCompleteInput():
+                global wordcomplete
```
Comment (rodja) [inline nicegui/elements/input.py]: Where is wordcomplete coming from?

Comment (rodja) [reply nicegui/elements/input.py]: Ah ok. You can have multiple PRs by creating separate branches on your forked repository.

Comment (rodja) [reply nicegui/elements/input.py]: > Btw, what do you think about that?

About what? Overall I think the auto completion is a great feature.

**Comment by falkoschindler** [comment]
@phoskee I think this is because you forked before we added `ui.darkmode`. Your fork is 43 commits behind our main. But that's normal and no problem at all. You can pull our main branch into yours at any time, or we do it later before merging.

**Comment by falkoschindler** [comment]
I'm a bit confused. The code changed by this PR looks identical to PR https://github.com/zauberzeug/nicegui/pull/824.
Should we close this one here without merging?

**Comment by falkoschindler** [comment]
Ok, I'll close this PR in favor of #837.

**Pull request closed without merging**